src/components/TimerModal/TimerSection.js

```diff
@@ -0,0 +1,14 @@
+import TimerResize from "./TimerResize";
+import TimerBundle from "./TimerBundle";
+import style from "./Modal.module.scss"
```

Comment:
Same as above, styling should not be done like this. Give components different classnames, follow the convention by using the global components as an example.